src/_guides/libraries/index.md

```diff
@@ -5,37 +5,41 @@ short-title: "Core libraries"
 toc: false
 ---
 
-Dart has a rich set of core libraries that provides essentials for many everyday
+Dart has a rich set of core libraries that provide essentials for many everyday
 programming tasks such as working on collections of objects,
 (`dart:collection`), making calculations (`dart:math`), and encoding/decoding
-data (`dart:convert`).
+data (`dart:convert`). Additional APIs are available in
+[community contributed packages](/guides/libraries/useful-libraries).
 
-The following table shows an inventory of all Dart core libraries. Aadditional
-APIs are available in community contributed packages located [in shared
-packages](/guides/libraries/useful-libraries). Also note that some core
-libraries are only available on some [Dart platforms](/platforms).
+The following table lists all of the Dart core libraries.
+Each library works on at least one [platform](/platforms):
+
+* **JIT:** The Dart VM in just-in-time compilation mode. Includes Flutter mobile, command-line, and server apps, unless otherwise noted.
```

Comment:
Please let's discuss these over in the platforms PR; I don't like the way these are phrased. So can we please remove them for now, work on them in the platforms PR, and then add back here (possibly via an include) when they are done.

Reply:
OK, will do. I'll edit #1505 to be a less invasive edit, perhaps with a TBD for the legend.